### ISSUE TITLE
Is it possible to have 3 or more column on wide screen ?

### ISSUE THREAD

**Issue by jeancharles-roger** [kawane/songbook#28]
On my laptop, the screen is really wide. Is it possible to have 3 or more column ?

**Comment by llgcode** [comment]
yes we can. Do you think it will be useful?

-- Laurent.

**Comment by jeancharles-roger** [comment]
Yeah in the screenshot, there is a bit of the lyrics on down scroll and there is a lot of space on the right.

I really think it could be nice.

![capture d ecran 2015-04-30 a 23 23 31](https://cloud.githubusercontent.com/assets/3531483/7422812/1ceb4c36-ef90-11e4-977a-9c99f7c6b4c1.png)

**Comment by llgcode** [comment]
sure.

-- Laurent.
